tests/test_schema.py

```diff
@@ -1003,6 +1003,23 @@ class MySchema(Schema):
         errors = s.validate({'foo': 42})
         assert errors['foo'] == [{'code': 'invalid_foo'}]
 
+    def test_ignored_if_not_in_only(self):
+        class MySchema(Schema):
+            a = fields.Field()
+            b = fields.Field()
+
+            @validates('a')
+            def validate_a(self, val):
+                raise ValidationError({'code': 'invalid_a'})
+
+            @validates('b')
+            def validate_b(self, val):
+                raise ValidationError({'code': 'invalid_b'})
+
+        s = MySchema(only=('b',))
+        errors = s.validate({'b': 'data'})
+        assert errors == {'b': {'code': 'invalid_b'}}
```

Comment:
By the way, I was expecting list in the value, like:
`{'b': [{'code': 'invalid_b'}]}`
Is it an issue?